presto-hive/src/test/java/com/facebook/presto/hive/TestShowStats.java

```diff
@@ -127,25 +122,25 @@ public void testShowStatsWithoutFromFails()
     @Test
     public void testShowStatsWithMultipleFromFails()
     {
-        assertQueryFails("SHOW STATS FOR (SELECT * FROM orders, lineitem)", ".*There must be exactly one table in query passed to SHOW STATS SELECT clause");
+        assertQueryFails("SHOW STATS FOR (SELECT * FROM orders, nation_partitioned)", ".*There must be exactly one table in query passed to SHOW STATS SELECT clause");
```

Comment:
Why this change?

Reply:
We need two tables to test testShowStatsWithMultipleFromFails()